### PR TITLE
Fix Newline

### PR DESCRIPTION
**Pull request by Maxboond**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+    count = 0
+
+    for i in phrase:
```
Comment (Dimosphen1) [inline app/main.py:19]: Do not use single-letter variable names in your code.

File: app/main.py [hunk 2]
```diff
@@ -14,4 +14,9 @@ def count_occurrences(phrase: str, letter: str) -> int:
...
+        if i.lower() == letter.lower():
+            count += 1
+    return count
```
Comment (Dimosphen1) [inline app/main.py:22]: You could try to implement this task in one line using built-in string methods and chaining to not allocate additional variables throughout your code execution.

**Review by Dimosphen1 — CHANGES_REQUESTED** [review]
Several changes were requested.

**Review by Maxboond — COMMENTED** [review]
fix newline

**Pull request closed without merging**